CardsCashCasino/Manager/TexasHoldEmManager.cs

```diff
@@ -0,0 +1,172 @@
+        BIG_BLIND,
+    }
+    
+    public class TexasHoldEmManager
+    {
+        /// <summary>
+        /// The cardManager for the game.
+        /// </summary>
+        private CardManager _cardManager;
+        
+        /// <summary>
+        /// The user's hand of cards.
+        /// </summary>
+        private UserHand _userHand;
+        
+        /// <summary>
+        /// Flag that indicates if the user is still active in the game.
+        /// </summary>
+        private bool _gameOver;
+        
+        /// <summary>
+        /// The chip manager for the game.
+        /// </summary>
+        private ChipManager _chipManager;
+        
+        /// <summary>
+        /// The main pot for the game.
+        /// </summary>
+        private int _mainPot;
+        
+        /// <summary>
+        /// The side pot for the game.
+        /// </summary>
+        private int _sidePot;
+        
+        /// <summary>
+        /// The current bet for the game.
+        /// This is the amount that all players must match to stay in the game.
+        /// </summary>
+        private int _currentBet;
+        
+        /// <summary>
+        /// The Small Blind for the game.
+        /// </summary>
+        private int _smallBlind;
+        
+        /// <summary>
+        /// The Big Blind for the game.
+        /// </summary>
+        private int _bigBlind;
+        
+        /// <summary>
+        /// The Ante for the game.
+        /// This is the amount that all players must pay to play the game.
+        /// </summary>
+        private int _ante;
+        
+        /// <summary>
+        /// A dictionary that tracks which player has what role.
+        /// </summary>
+        private Dictionary<PlayerRole, int> _players { get; set; }
+        
+        private List<CardHand> _playerHands { get; set; }
+
+        public TexasHoldEmManager()
+        {
+            _cardManager = new CardManager();
+            _userHand = new UserHand();
+            _gameOver = false;
+            _chipManager = new ChipManager();
+            _mainPot = 0;
+            _sidePot = 0;
+            _currentBet = 0;
+            _smallBlind = 0;
+            _bigBlind = 0;
+            _ante = 0;
+
+            // Initialize the player hands. The first player is the user.
+            for (int i = 0; i < Constants.AI_PLAYER_COUNT; i++)
+            {
+                if (i == 0)
+                {
+                    _playerHands[i] = new UserHand();
+                }
+                else
+                {
+                    _playerHands[i] = new PokerHand();
+                }
+            }
+            
+            // Initialize the player roles. The user starts as the dealer. 
+            _players = new Dictionary<PlayerRole, int>
+            {
+                {PlayerRole.DEALER, 0},
+                {PlayerRole.SMALL_BLIND, 1},
+                {PlayerRole.BIG_BLIND, 2}
+            };
+        }
+
+        public void StartNewGame()
```

Comment:
Few thoughts about this method:
1. Summary comment!
2. This seems incomplete OR we need to remove the white space at the bottom. Shouldn't there be something calling the first move of the game?
3. Clearing _playerHands and not regenerating it seems concerning.

Reply:
Wow. I completely neglected to finish this method. I'll make a wrapper function for creating the _playerHands list. and place it here